.github/workflows/check-handlebars-templates.yml

```diff
@@ -0,0 +1,39 @@
+name: Check handlebars templates
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  check-handlebars-templates:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+
+        - name: Read package.json node and npm engines version
+          uses: skjnldsv/read-package-engines-version-actions@v1.1
+          id: versions
+          with:
+            fallbackNode: '^12'
+            fallbackNpm: '^6'
+
+        - name: Set up node ${{ steps.versions.outputs.nodeVersion }}
+          uses: actions/setup-node@v2
+          with:
+            node-version: ${{ steps.versions.outputs.nodeVersion }}
+
+        - name: Set up npm ${{ steps.versions.outputs.npmVersion }}
+          run: npm i -g npm@"${{ steps.versions.outputs.npmVersion }}"
+
+        - name: Install handlebars
+          run: |
+            npm install -g handlebars
```

Comment:
Will always install latest. Maybe pin this? No idea how supported and maintained handlebar is nowadays

Reply:
It's the same in the old drone job.
The next touch to handlebar is to remove it and use vue 🤷‍♂️